cookbook/security/remember_me.rst

```diff
@@ -15,17 +15,20 @@ the session lasts using a cookie with the ``remember_me`` firewall option:
     .. code-block:: yaml
 
         # app/config/security.yml
-        firewalls:
-            default:
-                # ...
-                remember_me:
-                    key:      "%secret%"
-                    lifetime: 604800 # 1 week in seconds
-                    path:     /
-                    # by default, the feature is enabled by checking a
-                    # checkbox in the login form (see below), uncomment the
-                    # below lines to always enable it.
-                    #always_remember_me: true
+        security:
+            # ...
+
+            firewalls:
+                default:
+                    # ...
+                    remember_me:
+                        key:      "%secret%"
+                        lifetime: 604800 # 1 week in seconds
+                        path:     /
+                        # by default, the feature is enabled by checking a
+                        # checkbox in the login form (see below), uncomment the
+                        # below lines to always enable it.
```

Comment:
Instead of `the bellow lines`, we should say `the following line` or `the bellow line` because there is only one line.